<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSentenceDlg</class>
 <widget class="QDialog" name="AddSentenceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add/Remove Sentences</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>21</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>101</y>
     <width>87</width>
     <height>226</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="removeButton">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>158</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>94</y>
     <width>412</width>
     <height>352</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="sentenceLlabel">
      <property name="text">
       <string>Sentence</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2" colspan="3">
     <widget class="QTextEdit" name="sentenceTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>131</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>48</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>48</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="4">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>48</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="option1Label">
      <property name="text">
       <string>Option One</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="optionOneLineEdit"/>
    </item>
    <item row="3" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="option1Label_2">
      <property name="text">
       <string>Option Two</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="optionTwoLineEdit"/>
    </item>
    <item row="4" column="4">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="option1Label_3">
      <property name="text">
       <string>Option Three</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="optionThreeLineEdit"/>
    </item>
    <item row="5" column="4">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="option1Label_4">
      <property name="text">
       <string>Option Four</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="optionFourLineEdit"/>
    </item>
    <item row="6" column="4">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Option Five</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="optionFiveLineEdit"/>
    </item>
    <item row="7" column="3" colspan="2">
     <widget class="QPushButton" name="addSentenceButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>AddSentenceDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
